src/components/map/MapResults.tsx

```
@@ -0,0 +1,59 @@
+            <div>
+              <FilterDropdown
+                options={MAP_DROPDOWN_OPTIONS}
+                selected={selected}
+                onChange={value => setSelected(value)}
+              />
+            </div>
+          </div>
+          <div className={flex({ gap: 'lg' })}>
+            {data.map(place => (
+              <WideCard image={place.image}>
+                <WideCardContent
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)